hs_core/views/__init__.py

```diff
@@ -585,6 +585,11 @@ def create_new_version_resource(request, shortkey, *args, **kwargs):
     return HttpResponseRedirect(new_resource.get_absolute_url())
 
 
+@api_view(['POST'])
+def create_new_version_resource_public(request, pk):
+    return create_new_version_resource(request, pk)
```

Comment:
From the hs_restclient prespective I think it would be better if this returns the id of the new copied resource rather than the resource url.